joomla/joomla-cms
# Update jQuery Autocomplete to 1.4.2

### Summary of Changes
This PR updates [jQuery Autocomplete](https://github.com/devbridge/jQuery-Autocomplete) library to version [1.4.2](https://github.com/devbridge/jQuery-Autocomplete/releases/tag/v1.4.2) for **com_finder** and **mod_finder**.

### Testing Instructions
1. Apply patch
2. Test the **com_finder** and **mod_finder** typing any word in search field (for example, _joomla_  :wink:)
3. Make sure there are no JS errors and the list dropdown displayed correctly.

### Result
**com_finder**
![com_finder](https://cloud.githubusercontent.com/assets/3341562/16170122/fe4ed6e6-3550-11e6-9c32-6dc61b6d65be.png)

**mod_finder**
![mod_finder](https://cloud.githubusercontent.com/assets/3341562/16170130/1ceabb6a-3551-11e6-8d5c-1346241a66fd.png)
